### ISSUE TITLE
control which page elements display by default from /etc/defaults

### ISSUE THREAD

**Issue by holtzermann17** [ether/pad#69]
It's currently possible to change which page elements display by
commenting out undesired elements from src/templates/pad/pad_body2.ejs --
but it would be very user-friendly to provide options to add/remove these
in /etc/defaults -- e.g. to add or remove
TOPBAR
DOCBAR
BOTTOM AREA
and so forth,
and just generally to control the look/feel/layout of Pads in an easy way.

**Comment by redhog** [comment]
Hm, these could _either_ be if:ed out on the server, or better, just controlled by the current javascript controls, that is, using the above method to control the defaults for ?fullScreen=1&slider=0&sidebar=0

**Comment by holtzermann17** [comment]
Marking as high priority, because the non-modular nature of the UI is just going to be a problem for everyone until we get this fixed.

**Comment by redhog** [comment]
The UI is modular, in a way. The way you do this is with themes. You inherit a page in the theme and override files.

More javascript / url options control would be good thou.

Note about priority: Only set a high priority if you yourself have an inclination to fix this. This is not a wishlist, it's a todolist.

**Comment by holtzermann17** [comment]
Well calling this tag "Holtzermann17 todo" is really misleading.  I entered 1 zillion issues when we started this project, but they weren't all mine!  The tag was an attempt to zero in on the issues that actually seem working on _for anyone_.  Maybe I didn't get this categorization right.  But in any case I certainly didn't mean to "sign up" to work on all of the issues I tagged as high priority.  I wasn't thinking of it as a wishlist either.  (That's the set of all issues, really.)   So, yes, I do think of it as a todo list -- but rather than calling it _my_ todo list, how about a first draft of a 2.0?  I'll change the tag to something like that.
